data/en/components/radio/test_radio_mandatory.json

```diff
@@ -0,0 +1,39 @@
+{
+    "mime_type": "application/json/ons/eq",
+    "schema_version": "0.0.1",
+    "data_version": "0.0.2",
+    "survey_id": "0",
+    "title": "Other input fields",
+    "theme": "default",
+    "legal_basis": "StatisticsOfTradeAct",
+    "description": "A questionnaire to demo the mandatory radio field.",
+    "groups": [{
+        "blocks": [{
+                "type": "Questionnaire",
+                "id": "radio-mandatory",
+                "questions": [{
+                    "type": "General",
+                    "id": "radio-mandatory-question",
+                    "title": "radio-mandatory-question",
+                    "answers": [{
+                        "id": "radio-mandatory-answer",
+                        "label": "What is your favourite breakfast food",
+                        "mandatory": true,
+                        "options": [{
+                            "label": "None",
+                            "value": "None"
```

Comment:
We would never have a radio option with only one option, I think the tests should reflect that. I've added an issue - https://github.com/ONSdigital/eq-schema-validator/issues/6 - on the validator repo for this. If we're looking for a simple test case a Yes/No question would work well.